testsuite/features/step_definitions/common_steps.rb

```diff
@@ -647,7 +647,7 @@
   # Preparation of bootstrap script for different types of clients
   client = client_type == 'traditional' ? '--traditional' : ''
   # Use server if proxy is not defined as proxy is not mandatory
-  target = target_type.include? 'server' or $proxy.nil? ? $server : $proxy
+  target = (target_type.include? 'server' || $proxy.nil?) ? $server : $proxy
```

Comment:
Sorry, I just noticed that the end paren is too far, It must be:
`target = (target_type.include? 'server') || $proxy.nil? ? $server : $proxy`
In case Rubocop gets picky, the following is equivalent:
`target = if (target_type.include? 'server') || $proxy.nil? then $server else $proxy`

Reply:
Nothing to sorry about. I did a silly typo. Thanks for pointing me :-)